libp2p/Host.cpp

```diff
@@ -768,14 +640,38 @@ void Host::restoreNodes(bytesConstRef _b)
 		for (auto i: r)
 		{
 			auto id = (NodeId)i[2];
-			if (!m_nodes.count(id))
+			if (!m_peers.count(id))
 			{
 				bi::tcp::endpoint ep;
 				if (i[0].itemCount() == 4)
 					ep = bi::tcp::endpoint(bi::address_v4(i[0].toArray<byte, 4>()), i[1].toInt<short>());
 				else
 					ep = bi::tcp::endpoint(bi::address_v6(i[0].toArray<byte, 16>()), i[1].toInt<short>());
-				auto n = noteNode(id, ep, Origin::Self, true);
+//				auto n = noteNode(id, ep);
```

Comment:
This corresponds w/todos for implementing import/export of peers; to be implemented after all other code-review passes. (see follow-up comment regarding remaining tasks)